packages/synthetix-main/contracts/interfaces/IDelegation.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IDelegation {
+    enum Permission {
+        Owner,
+        Stake,
+        Unstake,
+        Mint,
+        Burn,
+        ClaimRewards,
+        ManageLocking,
+        Delegate
```

Comment:
permission to delegate is basically all the permission needed to be an owner (because you can just delegate permissions to yourself), right?

Reply:
you are right... I added it before checking the UI where I saw the "owner"